CMakeLists.txt

```diff
@@ -106,6 +106,25 @@ include(${PDAL_CMAKE_DIR}/options.cmake)
 
 set(PDAL_BUILD_TYPE ${CMAKE_BUILD_TYPE})
 
+# detect if we're setting up a visual studio solution
```

Comment:
This shouldn't go here.  If you want vsconfig.cmake in the cmake subdir, that's fine.

I don't know what this is about, but it looks pretty specific and subject to churn.

What does this file do?

Reply:
Unless you start Visual Studio from the command line in a conda environment, trying to run executables doesn't work because it can't find dependencies. This is just trying to preserve the path to the conda env so Visual Studio will set it in the environment when it runs the executable.

An alternative would be to set the working directory, but then pdal wouldn't see plugins unless they're installed to the conda env

Reply:
I get that the conda path isn't set when you start VS by clicking, but how does this help?  Just trying to understand VS, which I use almost never.

Reply:
The `user.props` file I'm making can be referenced/linked by a visual studio project, and will be the new defaults for the project settings unless specifically overridden by the project.

Specify the `LocalDebuggerEnvironment` will set environment variables, in this case I'm overriding the `PATH` variable to add the path to the dependencies (aka `%CONDA_PREFIX%/Libraries/bin`) to make them discoverable, kind of manual getting the conda environment setup.

I'm assuming the plugin install path `PDAL_PLUGIN_INSTALL_PATH` is the appropriate path here. But it may be worth adding a more specific cmake variable.